mscore/workspace.cpp

```diff
@@ -272,10 +273,16 @@ bool WorkspacesManager::isDefaultEditedWorkspace(Workspace* workspace)
 void WorkspacesManager::initCurrentWorkspace()
       {
       initWorkspaces();
-      m_currentWorkspace = findByName(preferences.getString(PREF_APP_WORKSPACE));
+
+      QString workspaceName = preferences.getString(PREF_APP_WORKSPACE);
+      m_currentWorkspace = findByName(workspaceName);
       Q_ASSERT(!workspaces().empty());
-      if (m_currentWorkspace == 0)
-            m_currentWorkspace = workspaces().at(0);
+      if (!m_currentWorkspace) {
+          m_currentWorkspace = findByTranslatableName(workspaceName);
+          if (!m_currentWorkspace) {
+              m_currentWorkspace = workspaces().at(0);
+          }
+      }
```

Comment:
For the benefit of an easier cherry-pick into master I'll let this coding style pass ;-)